sentry/templates/configmap-sentry.yaml

```diff
@@ -161,7 +161,16 @@ data:
     # information on configuring your queue broker and workers. Sentry relies
     # on a Python framework called Celery to manage queues.
     
+    {{- if .Values.rabbitmq.enabled }}
     BROKER_URL = os.environ.get("BROKER_URL", "amqp://{{ .Values.rabbitmq.username }}:{{ .Values.rabbitmq.password }}@{{ .Values.rabbitmq.host }}:5672//")
+    {{- else if .Values.redis.enabled }}
+      {{- if .Values.redis.password }}
```

Comment:
This is repeated, could you use a template helper instead?

Thanks.

Reply:
I changed this, since Redis is always enabled. Since I'm fairly new to helm charts, I'm not sure how to write helpers yet. But I could take a look at it.